src/Components/ToDoTask/index.js

```diff
@@ -47,7 +47,7 @@ export function ToDoTask({ task, editTodo, removeTodo }) {
             }}
           />{" "}
```

Comment:
why do we need {" "}?

Reply:
These are included by prettier extension.